File: src/main/java/org/dedda/bratwurst/lang/Exit.java
package org.dedda.bratwurst.lang;

import org.dedda.bratwurst.lang.scope.Scope;

/**
 * Created by dedda on 10/14/15.
 *
 * @author dedda
 */
public class Exit extends BWInstruction {

    public Exit(int lineNumber) {
        super(lineNumber);
    }

    @Override
    public void run(Scope scope) {
        Program.getInstance().stop(0);
    }
}

File: src/main/java/org/dedda/bratwurst/lang/Exit.kt
package org.dedda.bratwurst.lang

import org.dedda.bratwurst.lang.scope.Scope

/**
 * Created by dedda on 10/14/15.
 *
 * @author dedda
 */
class Exit(lineNumber: Int) : BWInstruction(lineNumber) {
    override fun run(scope: Scope) {
        scope.program.stop(0)
    }
}